plugins/copr.py

```diff
@@ -472,28 +474,30 @@ def _download_repo(self, project_name, repo_filename):
         api_path = "/coprs/{0}/repo/{1}/dnf.repo?arch={2}".format(project_name, short_chroot, arch)
 
         try:
-            f = self.base.urlopen(self.copr_url + api_path, mode='w+')
-        except IOError as e:
+            response = urlopen(self.copr_url + api_path)
             if os.path.exists(repo_filename):
                 os.remove(repo_filename)
-            if '404' in str(e):
-                try:
-                    res = urlopen(self.copr_url + "/coprs/" + project_name)
-                    status_code = res.getcode()
-                except HTTPError as e:
-                    status_code = e.getcode()
-                if str(status_code) != '404':
-                    raise dnf.exceptions.Error(_("This repository does not have"
-                                                 " any builds yet so you cannot enable it now."))
-                else:
-                    raise dnf.exceptions.Error(_("Such repository does not exist."))
-            raise
-
-        for line in f:
-            if re.match(r"\[copr:", line):
-                repo_filename = os.path.join(self.base.conf.get_reposdir,
-                                             "_" + line[1:-2] + ".repo")
-            break
+        except HTTPError as e:
+            error_msg = _("It wasn't possible to enable this project.\n")
+            error_msg = _(''.join(
+                [error_msg,
+                 "Chroot {0} does not exist in {1}.\n".format('-'.join(self.chroot_parts), project_name)]))
+            error_data = e.headers.get("Copr-Error-Data")
+            if error_data:
+                error_data_decoded = base64.b64decode(error_data).decode('utf-8')
+                error_data_decoded = json.loads(error_data_decoded)
+                if len(error_data_decoded["available chroots"]) != 0:
+                    error_msg = _(''.join([error_msg, "Available chroots: {0}".format(
```

Comment:
the `_()` method needs to have a constant, so please move it around the `"Available chroots: {0}"` string directly.